### ISSUE TITLE
Cryptographic Challenge from Server

### ISSUE THREAD

**Issue by elprl** [hansemannn/iOS16-Passkeys-Sample#1]
I noticed you manually created the challenge using a random set of numbers. This is supposed to come from a server right? 
What's the trade-off to it being manually created here?

Any idea how to setup this up on something like Firebase or Lambda cloud functions?

**Comment by hansemannn** [comment]
Yes, it should definitely come from the server. I was not able to continue with the server so far, so I will keep it open for now.
